ios/RCTContacts/RCTContacts.m

```diff
@@ -708,17 +708,21 @@ -(void) updateRecord:(CNMutableContact *)contact withData:(NSDictionary *)contac
 
 + (NSData*) imageData:(NSString*)sourceUri
 {
-    if ([sourceUri isEqualToString:@"LOCALL_CONTACT"]) {
-        sourceUri = [[NSBundle mainBundle] pathForResource:@"locall_contact" ofType:@"png"];
-        //NSLog(@"Contact image uri %@", sourceUri);
-    }
-
+    NSURL *url = [NSURL URLWithString:sourceUri];
+    
     if([sourceUri hasPrefix:@"assets-library"]){
```

Comment:
Here is a check for assets-library, couldn't this be used for loading the image instead of what you do in the else clause to load from the main bundle?

Reply:
As I understand it assets-library is the camera roll in iOS

Reply:
Ok, never mind then.